src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -292,7 +293,7 @@ public static void removeRequestFromResolver(final IStandardRequestManager manag
      *
      * @throws IllegalArgumentException when the token is unknown or the request is not assigned yet.
      */
-    public static IRequestResolver getResolverForRequest(final IStandardRequestManager manager, final IToken<?> requestToken)
+    public static IRequestResolver<? extends IRequestable> getResolverForRequest(final IStandardRequestManager manager, final IToken<?> requestToken)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)